fullstack-cert/react-projects/tic-tac-toe/user-stories.md

```diff
@@ -0,0 +1,14 @@
+1. You should create and export a `Square` component that accepts two props: `value` and `onClick`.
```

Comment:
we will probably need to remove all user stories pointing to export child components into a parent component file because I don't think our current system is setup for that.

looking at the current projects, there is only one `index.jsx` file and both parent and child components live there
https://www.freecodecamp.org/learn/full-stack-developer/lab-mood-board/build-a-mood-board

For now, lets remove all user stories about exporting child components. 

```suggestion
1. You should create a `Square` component that accepts two props: `value` and `onClick`.
```